video.go

```diff
@@ -45,7 +45,8 @@ type Video struct {
 	CompanionAds    []Banner            `json:"companionad,omitempty"`
 	APIs            []APIFramework      `json:"api,omitempty"` // List of supported API frameworks
 	CompanionTypes  []CompanionType     `json:"companiontype,omitempty"`
-	Placement       VideoPlacement      `json:"placement,omitempty"` // Video placement type
+	Placement       VideoPlacement      `json:"placement,omitempty"` // Video placement type, DEPRECATED
+	Plcmt 			VideoPlcmt 			`json:"plcmt,omitempty"`	// Video Plcmt type ad defined in ADCOM1.0
```

Comment:
sorry, but could you pls run `go fmt` on those files? https://github.com/bsm/openrtb/blob/a7bad4884ea390485f44262b9577c1814055f53b/video.go#L49

Reply:
Done.